### DISCUSSION TITLE
Script to remove Google search redirects and tracking?

### DISCUSSION THREAD

**Discussion by ghost** [quoid/userscripts#284] [General]
Hello - just wondering if anyone has a userscript that is known to work with Safari/iOS, for the purpose of removing redirects and tracking events (click handlers) from Google search results?

Originally I was using the source code to the "Don't Track Me Google" Chrome extension [0] and I thought that was working but as it turns out, that was not working... Then I tried "Google Tracking B Gone" [1] and as far as I can see that doesn't work either.

- [0] https://github.com/Rob--W/dont-track-me-google/blob/master/contentscript.js
- [1] https://github.com/stacybrock/google-tracking-b-gone/blob/master/google-tracking-b-gone.user.js

**Comment by ghost** [comment]
Not sure what I was doing wrong originally, but the script from link [0] above has been working for a while now with just the addition of the following header at the top:

```js
// ==UserScript==
// @name         AntiGoogle
// @match        *://*.google.com/*
// @version      1.0
// ==/UserScript==
```

**Reply by quoid** [reply]
glad its working!

**Comment by ACTCD** [comment]
Here's a simple user script that provides such a feature, it's designed for this extension and works well, but I'm sure it won't provide as many features as you mentioned in [0].

https://github.com/ACTCD/Userscripts/blob/main/userjs/ACT.Google.DM.Trackless.user.js
